staging/src/k8s.io/dynamic-resource-allocation/structured/allocator.go

```diff
@@ -1071,68 +1095,103 @@ func taintTolerated(taint resourceapi.DeviceTaint, request requestAccessor) bool
 	return false
 }
 
-func (alloc *allocator) checkAvailableCapacity(device deviceWithID) (bool, error) {
+// checkAvailableCounters checks if there are enough counters available to allocate
+// the specified device.
```

Comment:
```suggestion
// the specified device.
//
// Gets called only if the partitionable devices feature is enabled and the device
// consumes counters.
```

Reply:
I was wondering whether the function should check this and had to go back to the caller to find out.